### PR TITLE
GREET Data Hotfix

### PR DESCRIPTION
**Pull request by dakotaramos**
Hotfix to correct values and units for iron ore mining and pelletizing emission intensities based on feedback from ANL.

## Related issue
https://github.com/NREL/HOPP/pull/380

## Impacted areas of the software
resources

## Additional supporting information
This work is being completed for the LCA integration, the data pulling and parsing will live in HOPP, the LCA calculations with that data will live in GreenHEART

## Test results, if applicable
tests/hopp/test_greet_resource.py ...

### CODE REVIEW

**Comment by johnjasa** [comment]
Thanks for this, @dakotaramos! The changes you've made look good.
The failing CI appears unrelated to your changes. @RHammond2, could you please take a look and see if a package update or similar has caused these tests to fail?

**Comment by RHammond2** [comment]
> Thanks for this, @dakotaramos! The changes you've made look good. The failing CI appears unrelated to your changes. @RHammond2, could you please take a look and see if a package update or similar has caused these tests to fail?

The issue is stemming from the `KML` object having `features` as a callable, so if we change [these lines](https://github.com/dakotaramos/HOPP/blob/8a3b94a430c91def08c7b75d676b07bf373a9cb4/hopp/simulation/technologies/sites/site_info.py#L253-L254) to use `k.features` instead of `k.features()`, then we should be ok in theory.

However, something I noticed is that we'll also be attempting to index an empty list in the tests, so this will raise a new error. Doubly however, this is call was not previously an issue, so I checked out the `fastkml` changelog, and they had a v1.0 release in mid-November.

tl;dr: Pin fastkml to be < 1, and open an issue to migrate HOPP and GreenHEART to be compatible with fastkml v1.0.

**Review by RHammond2 — APPROVED** [review]
Aside from the `fastkml` issue in my comment this all looks good to me.

**Comment by dakotaramos** [comment]
@RHammond2 @johnjasa just pushed the update to the fastkml version in pyproject.toml

**Review by johnjasa — APPROVED** [review]
